pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -376,6 +376,9 @@ func getMarkerScanners(logsCommandName, securityPolicyCommandFormat, setProbeCom
 		func(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 			return deployanalysis.FindDeploymentConfigReadinessWarnings(g, f, setProbeCommandName)
 		},
+		func(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
+			return deployanalysis.FindDeploymentConfigLivenessWarnings(g, f, setProbeCommandName)
```

Comment:
Shouldn't we also be marking pods that are not under a DC? Lone pods, petsets, pods from RCs